pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -206,6 +210,9 @@ func (m *manager) AddContainer(pod *v1.Pod, container *v1.Container, containerID
 
 		m.policyRemoveContainerByRef(string(pod.UID), initContainer.Name)
 	}
+
+	// the pod passed the admit phase and we can remove the pod from the admitted pods
+	delete(m.admittedPods, string(pod.UID))
```

Comment:
this seems unnecessary - cleanup is performed in `updatedAdmittedPods` anyway but should be harmless as well